packages/opentelemetry-core/src/trace/sampler/ProbabilitySampler.ts

```diff
@@ -31,9 +27,11 @@ export class ProbabilitySampler implements Sampler {
   }
 
   toString(): string {
-    return `${this.description}{${this._probability}}`;
+    if (this._probability >= 1.0) return 'AlwaysSampleSampler';
+    else if (this._probability <= 0) return 'NeverSampleSampler';
+    return `ProbabilitySampler{${this._probability}}`;
```

Comment:
What I meant in my previous comment is to always use this string and not the previous 2 lines. There is no benefit to distinguish between `AlwaysSampleSampler` and `NeverSampleSampler`.

Reply:
+1 for simple code when we can do so (less to ship and parse in the browser)

Reply:
I agree with simplicity, but these names are mentioned in the specs: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sampling-api.md#built-in-samplers.

Always sample -> Description MUST be `AlwaysSampleSampler`.
Never sample -> Description MUST be `NeverSampleSampler`.

thoughts?

Reply:
What is the rationale? If it's specific to Java we can ignore this.

Reply:
I think these names are clear and simple to understand when you see the debug pages and the logs. For now. I can add TODO to consider to use these names later once we have agreed.